srcjs/notifications.js

```diff
@@ -0,0 +1,160 @@
+exports.notifications = (function() {
+
+  // Milliseconds to fade in or out
+  const fadeDuration = 250;
+
+  function show({ html=null, duration=5000, id=null, closeButton=true,
+                  style = null } = {})
```

Comment:
I think @jcheng5 has a good point. Currently, some of the Shiny inputs can take a `style` argument (actually passed in through `...`) that lets you set any CSS thing... For example, this is totally valid (taken from a [stackoverflow answer](http://stackoverflow.com/a/35871042)):

```{r}
actionButton("run", "Run Analysis", icon("paper-plane"), 
             style="color: #fff; background-color: #337ab7; border-color: #2e6da4")
```

So, `style` might be confusing... I think using `class` would be okay... App authors are probably familiar with the bootstrap classes already...